run_examples/run_ogcore_example.py

```diff
@@ -73,6 +73,8 @@ def main():
     )
     # Update parameters for baseline from default json file
     p.update_specifications(og_spec)
+    # make e matrix 3D
```

Comment:
@jdebacker. Shouldn't we make the default e matrix 3-d in `default_parameters.json` lines 170-278? We should change the `number_dims` to 3 in line 177. And we should add one more set of brackets around the input `value` to make it a three dimensional nested list. Then we just need to code `parameters.py` to copy any two dimensional input down the t-dimension--as would be the case in the default specification. What do you think?

Reply:
@rickecon At some point, we might.  But my thought is to keep this "under the hood" for now.  If we change `default_parameters.json` then we need to update each country repo with the 3D `e`.  

Since we're not sure how many use cases there will be of this, I would say we leave `default_parameters.json` as is, then when one wants to update to a 3D `e`, they just do so outside of the `update_specfications` method (i.e., they just assign `p.e = new_array` with a `new_array` that is of shape `(p.T, p.S, p.J)`.